testnet2/testnet2.toml

```diff
@@ -6,28 +6,28 @@ Key = "network"
 Value = "testnet2"
 
 [API]
-
   [API.GRPC]
     Hosts = [
       "api-n00.validators-testnet.vega.rocks:3007",
       "api-n02.validators-testnet.vega.rocks:3007",
       "api-n03.validators-testnet.vega.rocks:3007",
       "api-n04.validators-testnet.vega.rocks:3007",
       "api-n05.validators-testnet.vega.rocks:3007",
-      "vega-testnet.anyvalid.com:3007"
+      "vega-testnet.anyvalid.com:3007",
+      "testnet.grpc.vega.xprv.io:443"

```

Comment:
This one also doesn't work for my client, and I don't get any specific error. @ValentinTrinque are you able to check this endpoint?

Reply:
I spoke to XPRV validators. I will have a look to secure client to verify the origin of the problem.

https://github.com/vegaprotocol/vega/issues/7834